executor/brie.go

```diff
@@ -172,7 +260,8 @@ func (b *executorBuilder) buildBRIE(s *ast.BRIEStmt, schema *expression.Schema)
 	e := &BRIEExec{
```

Comment:
Can we split buildBRIE into multiple small functions, like buildBackup, buildRestore, buildImport